src/components/tests/__snapshots__/QuickOpenModal.spec.js.snap

```diff
@@ -1458,14 +1458,14 @@ exports[`QuickOpenModal updateResults on enable 2`] = `
     >
       <Modal
         handleClose={[Function]}
-        status="entering"
+        status="exited"
```

Comment:
Quick open slides in and slides out. It is also fine in the other 10 unit tests. 

I am going to move forward and we can figure this mystery out later. I believe it is just a timing thing.